code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -200,13 +200,17 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 			++ind
 			if(ind > 1)
 				. += ", "
-			. += "\tSynthetic [organ_name]"
-		else if(status == "digital")
+			switch(organ_name)
+				if ("brain")
+					. += "\tPositronic [organ_name]"
+				else
+					. += "\tSynthetic [organ_name]"
+		else if(status == "drone")
 			++ind
 			if(ind > 1)
 				. += ", "
 			. += "\tDigital [organ_name]"
-		else if(status == "assisted")
+		else if(status == "cybernetic")

```

Comment:
Oh yeah, I missed that one and another one. Fixed now. Went through the file another time to be sure.